src/main/java/com/example/springonlinebookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,39 @@
+package com.example.springonlinebookstore.dto.user;
+
+import com.example.springonlinebookstore.annotation.FieldsValueMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+@Setter
+@Getter
+@EqualsAndHashCode
+@RequiredArgsConstructor
+@FieldsValueMatch.List({
+        @FieldsValueMatch(field = "password",
+                fieldMatch = "repeatPassword",
+                message = "The password fields must match")})
```

Comment:
Why so complicate usage of annotation? make something like this:
```suggestion
@FieldsValueMatch(
        field = "password",
                fieldMatch = "repeatPassword",
                message = "The password fields must match")
                )
```